src/EnergyPlus/SizingManager.cc

```diff
@@ -770,6 +782,778 @@ namespace SizingManager {
+					Real64 calcSysAirMinFlowRat( 0.0 );
+					if ( FinalSysSizing( AirLoopNum ).DesMainVolFlow > 0.0 ) { // protect div by zero
+						calcSysAirMinFlowRat = FinalSysSizing( AirLoopNum ).DesHeatVolFlow / FinalSysSizing( AirLoopNum ).DesMainVolFlow;
+					}
+					ReportSizingManager::ReportSizingOutput( "AirLoopHVAC", curName, "Calculated Heating Air Flow Ratio []", calcSysAirMinFlowRat );
+					OutputReportPredefined::PreDefTableEntry( OutputReportPredefined::pdchSysSizCalcHeatFlowRatio, curName, calcSysAirMinFlowRat, 4 );
+				}
+			}
+		}
+
+	} // if doing any system sizing
+	}
+
+	void
+	ManageSystemVentilationAdjustments()
+	{
+		// redo std 62.1 calculations using latest information on zone flows and report to tables
+
+		//redo 62.1 zone calculations with final (or user) zone terminal flow sizes, only redo calculations that might change with final flows
+		for ( int AirLoopNum = 1; AirLoopNum <= NumPrimaryAirSys; ++AirLoopNum ) {
+			int SysSizNum = InputProcessor::FindItemInList( FinalSysSizing( AirLoopNum ).AirPriLoopName, SysSizInput, &SystemSizingInputData::AirPriLoopName );
+			if ( SysSizNum == 0 ) SysSizNum = 1; // use first when none applicable
+			if ( FinalSysSizing( AirLoopNum ).OAAutoSized && SysSizInput( SysSizNum ).SystemOAMethod == SOAM_VRP && DataAirLoop::AirLoopZoneInfo( AirLoopNum ).NumZones > 1 && FinalSysSizing( AirLoopNum ).LoadSizeType != Ventilation ) {
+
+				// Loop over all zones connected to air loop, redo both cooling and heating calcs for Zdz minimum discharge outdoor air fraction for each zone
+				for ( int zoneNumOnLoop = 1; zoneNumOnLoop <= DataAirLoop::AirLoopZoneInfo( AirLoopNum ).NumZones; ++zoneNumOnLoop  ) {
+					int CtrlZoneNum = DataAirLoop::AirLoopZoneInfo( AirLoopNum ).ActualZoneNumber( zoneNumOnLoop );
+					if ( DataSizing::VdzMinClgByZone( CtrlZoneNum ) > 0.0 ) {
+						DataSizing::ZdzClgByZone( CtrlZoneNum ) = min( 1.0, FinalZoneSizing( CtrlZoneNum ).VozClgByZone / DataSizing::VdzMinClgByZone( CtrlZoneNum ) );
+					} else { // would divid by zero, so set to max ??
+						DataSizing::ZdzClgByZone( CtrlZoneNum ) = 1.0;
+					}
+					if ( DataSizing::VdzMinHtgByZone( CtrlZoneNum ) > 0.0 ) {
+						DataSizing::ZdzHtgByZone( CtrlZoneNum ) = min( 1.0, FinalZoneSizing( CtrlZoneNum ).VozHtgByZone / DataSizing::VdzMinHtgByZone( CtrlZoneNum ) );
+					} else { // would divid by zero, so set to max
+						DataSizing::ZdzHtgByZone( CtrlZoneNum )  = 1.0;
+					}
+				} // end loop over zones on air loop to calculate Zdz values
+
+				//Sum Voz values for System Vou, in E+ the Vbz value has now been corrected to remove population Diversity, so we add the term back in here directly to get Vou
+				DataSizing::VouBySys( AirLoopNum ) = 0.0;
+				for ( int zoneNumOnLoop = 1; zoneNumOnLoop <= DataAirLoop::AirLoopZoneInfo( AirLoopNum ).NumZones; ++zoneNumOnLoop  ) {
+					int CtrlZoneNum = DataAirLoop::AirLoopZoneInfo( AirLoopNum ).ActualZoneNumber( zoneNumOnLoop );
```

Comment:
This and next for loop are looping through the same zones. Should have combined the calculations.